runtime/pack/dist/opt/termdebug/plugin/termdebug.vim

```diff
@@ -1715,7 +1715,7 @@ def GotoVariableswinOrCreateIt()
       exe $'resize {GetVariablesWindowHeight()}'
     endif
   else
-    win_gotoid(varwins[0])
+    win_gotoid(varwin)
   endif
 
   if running && !empty(win_findbuf(varbufnr))
```

Comment:
```suggestion
  if running && bufwinid(varbufname) > 0
```

I didn't look for other uses of `win_findbuf`, but this one was just sitting there visible when I looked at the patch.

Reply:
Done! I used `bufname `as per `:h` :)

Reply:
You might take a look at `:help bufname`, that string is file-pattern plus some extras and there's some weirdness. Using a number as the arg is unambiguous; but I doubt it matters here.

Reply:
Oh ok! I never noticed that. Good to know!